### PR TITLE
add explicit error message when kuzzle is unreachable

### PR DESCRIPTION
**Pull request by stafyniaksacha**

### CODE REVIEW

**Comment by dbengsch** [comment]
+1

**Comment by benoitvidis** [comment]
+1

File: src/kuzzle.js [hunk 1]
```diff
@@ -321,11 +321,15 @@ Kuzzle.prototype.connect = function () {
   });
 
   self.socket.on('connect_error', function (error) {
+    var connexionError = new Error('Unable to connect to kuzzle server at "' + self.url + '"');
```
Comment (scottinet) [inline src/kuzzle.js]: Nitpicking: in english it's `connection`. :-)

**Comment by jenow** [comment]
+1

**Pull request merged**